src/clients/github.js

```diff
@@ -166,9 +147,11 @@ export function initAddCommitAndPush(username, projectName, isTwoFactorAuth) {
         .init()
         .add('.gitignore')
         .add('./*')
-        .commit('Initial commit');
-      console.log('We cannot push hammer-io generated code to your repository because you have 2fa enabled. ' +
-        'Please follow this link (https://help.github.com/articles/providing-your-2fa-authentication-code/) for support.');
+        .commit('Initial Commit w/ :heart: by @hammer-io.');
+      console.log(chalk.red('We cannot push hammer-io generated code to your repository because' +
+      ' you have 2fa enabled. ' +
+        'Please follow this link' +
+        ' (https://help.github.com/articles/providing-your-2fa-authentication-code/) for support.'));
```

Comment:
What are they supposed to do after visiting that link? Should we tell them to push manually with the git commands? Should they run tyr again?